index.rst

```diff
@@ -326,33 +326,167 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
+  WHERE v.band='r'
+  AND c.seeing < 0.7
+  AND c.skyBg > 21
+  AND v.airmass < 1.2
+  AND v.expTime = 30;
+
+
+**Telemetry summary tables**
+
+The current approach for combining visit information with telemetry data is a "join on time," which essentially involves querying two databases, the Butler registry to retrieve the visit's start and end times and the EFD to retrieve the telemetry values in that time range.
+
+The database `use cases`_ confluent page indicates the telemetry information that would be most useful in the Visit database.
+Typically the full time series from the EFD is not required and should consolidate the information (usually spread across multiple EFD topics) into fewer tables to make it easier to query and join with other tables in the Visit database.
+
+A tool for creating these "telemetry summary tables" in a relational database does not currently exist.
+The `SQR-58 The EFD transformation service`_ presents a possible solution by filtering and combining information from multiple EFD topics and aggregating values over time.
+This tool must be flexible enough to incrementally add new tables or add new columns to existing tables in the Visit database as needed.
+For the Rapid Analysis use case, however, loading past data into the new tables/columns is not required after evolving the schema.
+
+Note that Camera diagnostic metrics are currently recorded in a separate database.
+We recommend recording that in the Visit database as well.
+
+In addition, we could also replicate visit IDs and visit start/end times from the Butler registry into the EFD to enable the "join on time" with a single query in the EFD.
+
+**Exposure and narrative log tables**
+
+A system for exposure-level and human narrative input is in place through the `exposure log`_ and `narrative log`_ tools.
+The corresponding tables should exist in the Visit database to allow joins with other tables in the Visit database.
+
+**Scheduler tables**
+
+Currently, the only way to retrieve scheduler snaphost files is by querying the ``logevent_largeFileObjectAvailable`` event recorded in the EFD.
+That topic contains the URI to retrieve the corresponding file from the LFA.
+
+Adding a table in the Visit database linking the scheduler snapshot files with the observations would make those files more accessible to tools like the schedview (see `RTN-037 Architecture for Scheduler and Observing Progress Monitoring Software`_ ).
+
+**Calibration Products table**
+
+A table for keeping track of the calibration set used in a given Rapid Analysis run and the time when the calibration set was updated is important for correlating those changes with image quality metrics.
+
+Currently, for AuxTel, the calibration collection in the Butler is updated manually, and the list of calibrations used in a given run is stored in the output headers (see DM-37129).
+
+The Butler provenance system might also help to solve this problem (see `DMTN-205 Tracking Provenance in Butler`_).
+
```

Comment:
This section misses the calibration discussion in my DB confluence page.  We want to be able to use a calibrations database to track the state of the system (e.g. the read noise per amplifier), so as a function of time (which means calibration dataset).

We need this information at the summit to monitor the health of the camera, but as for the visit DB we also need it at USDF.

Reply:
I thought that was part of the camera diagnostics information, but looking again, that also results from the calibration pipeline outputs and RA. I've said it more explicitly in the calibration settings:
```
In addition, we want to be able to store results from the calibrations pipeline and Rapid Analysis to track the state of the system (e.g., the read noise per amplifier) and so a function of time. 
We need this information at the Summit to monitor the camera's health. 
```